notebooks/2.1.1-he-rain-analysis.ipynb

```diff
@@ -72,22 +72,31 @@
    "metadata": {},
```

Comment:
<p>this just station data all by itself (station data vs station data)</p><p>testing to see if there is that much difference between the "truth" from the different station metrics (ie, consensus, consensus amongst the same ecosystem)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/DataKind-DC/audubon-cbc/pull/68/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='157'/>

Reply:
<p>Fixed in Rain, snow, and snowfall notebooks</p>